### ISSUE TITLE
Account connections aren't working

### ISSUE THREAD

**Issue by anarisis** [Cloud-Player/web#72]
Account connections haven't been working for about two weeks now for either soundcloud or youtube, making it impossible to access playlists.

**Comment by Zarlex** [comment]
What do you mean with does not work? Are you unable to connect with your soundcloud/youtube account? Were you able to connect your soundcloud/youtube account with Cloud-Player before?

**Comment by anarisis** [comment]
That is correct. I was previously able to connect via soundcloud but connecting now doesn't load any of my playlists on either web or desktop.

**Comment by Zarlex** [comment]
Is it still not working for you?
